### DISCUSSION TITLE
[Feature Suggestion] Package Auto Update

### DISCUSSION THREAD

**Discussion by Dane411** [Devolutions/UniGetUI#461] [General]
Hi, 

I've just realized that there are some packages that are updated very regularly, ie. Java or VC++ Redistributables, so I thought it would be nice if you'd consider to add the ability to flag those packages to auto update without asking the user.

Thank you in advance

**Comment by marticliment** [comment]
No, because theyrequire user ionteraction to get updated

**Reply by ppvnf** [reply]
Would it be possible to have an option to make clicking the update notification be able to perform the update without additional user interaction?

**Reply by marticliment** [reply]
No, Win32 desktop apps do not allow notification interaction

**Reply by Dane411** [reply]
I was involved in the development of a software discovery and upgrade solution, and as far as I remember, this package can be silently installed without user interaction.
According to a quick Google search it seems so.
Not sure if with the following command: vcredist_x64.EXE /Q

**Reply by marticliment** [reply]
Yes, but winget does not handle that very well. Some packages just don't work, some other crash, some other require administrator rights and will interrupt the user and some just ignore the flag. This was already discussed, and trust me, i have tested every single way possible
